src/commands/up.rs

```diff
@@ -127,7 +131,13 @@ impl UpCommand {
                     .as_deref()
                     .unwrap_or_else(|| DEFAULT_MANIFEST_FILE.as_ref());
                 let bindle_connection = self.bindle_connection();
-                spin_loader::from_file(manifest_file, &working_dir, &bindle_connection).await?
+                spin_loader::from_file(
+                    manifest_file,
+                    &working_dir,
+                    &bindle_connection,
+                    self.components,
```

Comment:
What is the error behaviour if all components are excluded?  For example, if I have an app with `hello` and `goodbye` components, and pass `-c goobdye`?  Does it tell me there are no components, or does it report specifically that no components matched the include arguments?

Thinking about it, passing an id that doesn't exist in the manifest should probably be an error.  I can't think of any situation in which that would be what the user wanted...?